Cargo.toml

```diff
@@ -179,39 +179,43 @@ codegen-units = 1
 rpath = true
 
 [patch.crates-io]
-# Fork and rename to use "OG" dalek-cryptography.
-bulletproofs-og = { git = "https://github.com/mobilecoinfoundation/bulletproofs.git", rev = "675330c754f28876dbf94fc303fe73666cf8f8f4" }
+# Fork to use latest dependencies.
+bulletproofs = { git = "https://github.com/remoun/bulletproofs.git", branch = "mobilecoin" }
 
 # This version contains iOS build fixes
 cmake = { git = "https://github.com/alexcrichton/cmake-rs", rev = "5f89f90ee5d7789832963bffdb2dcb5939e6199c" }
 
 # Fix issues with recent nightlies
-datatest = { git = "https://github.com/mobilecoinfoundation/datatest.git", rev = "dc3521f692228b846a2312e7ceece50e7664be6e" }
+datatest = { git = "https://github.com/mobilecoinfoundation/datatest.git", branch = "mobilecoin" }
```

Comment:
Please pin to revisions rather than branches so that we can have repeatable builds
 If we cannot repeatably build the enclave then it makes remote attestation worthless. Branches are lightweight and can be changed